dominos/src/test/java/me/fontys/semester4/dominos/pizza/and/ingredient/PizzaAndIngredientConverterTest.java

```diff
@@ -0,0 +1,45 @@
+package me.fontys.semester4.dominos.pizza.and.ingredient;
+
+import me.fontys.semester4.data.entity.Category;
+import me.fontys.semester4.data.entity.Ingredient;
+import me.fontys.semester4.data.entity.Product;
+import me.fontys.semester4.data.entity.ProductPrice;
+import org.junit.Test;
+import java.io.FileNotFoundException;
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+
+//public class PizzaAndIngredientConverterTest {
+//    private String fileName = "C:\\Users\\884573\\Documents\\Repositories\\_Semester_4_ImportApp\\dominos\\src\\test\\resources\\pizza_ingredienten_test.csv";
+//    private char separator = ';';
+//
```

Comment:
If it's not working, or not being used; don't commit it. Only commit functional stuff that is actually being used.

Reply:
You're right of course. And very strict. Push inc.